libairspy/src/iqconverter_float.c

```diff
@@ -102,7 +117,7 @@ void iqconverter_float_reset(iqconverter_float_t *cnv)
 	memset(cnv->fir_queue, 0, cnv->len * sizeof(float) * SIZE_FACTOR);
 }
 
-static _inline float process_fir_taps(const float *kernel, const float *queue, int len)
+static __inline float process_fir_taps(const float *kernel, const float *queue, int len)
```

Comment:
Keep old code with _inline and defined #define _inline to what is supported by FreeBSD it seems to be __inline like it is done for other Platform/Compiler ...